src/BunyanDebugStream.coffee

```diff
@@ -224,8 +223,8 @@ class BunyanDebugStream extends Writable
             #{date}#{processStr}#{levelPrefix}#{src}#{prefixes}#{applyColors message, colorsToApply}
         "
 
-        line += "\n#{INDENT}#{request}" if request?
-        line += "\n" + applyColors(values.join('\n'), colorsToApply) if values.length > 0
+        line += "\n#{@_indent}#{request}" if request?
+        line += "\n" + values.map((v) -> applyColors(v, colorsToApply)).join('\n') if values.length > 0
```

Comment:
Where are you running into trouble with colors?  The old version should work fine in a regular terminal.